src/lib/route-path.ts

```diff
@@ -25,8 +25,10 @@ export class RoutePath {
   static readonly USER_DEFAULT_LOGGED_IN_PAGE = RoutePath.USER_MY_PETS;
 
   static readonly VET_LOGIN_PAGE = "/vet/login";
-  static readonly VET_DASHBOARD_PAGE = "/vet/dashboard";
-  static readonly VET_DEFAULT_LOGGED_IN_PAGE = RoutePath.VET_DASHBOARD_PAGE;
+  static readonly VET_SCHEDULE_APPOINTMENTS = "/vet/schedule-appointments";
+  static readonly VET_ADD_REPORTS = "/vet/add-reports";
```

Comment:
Maybe change this to /vet/schedule and /vet/report?

Reply:
Hmm... I think it is okay to leave it as is first. Let's revisit when the development is more mature.

Reply:
Done! - fc776df